pkgs/applications/networking/cluster/argocd/default.nix

```diff
@@ -28,13 +28,15 @@ buildGoModule rec {
       "-s" "-w"
       "-X ${package_url}.version=${version}"
       "-X ${package_url}.buildDate=unknown"
-      "-X ${package_url}.gitCommit=${commit}"
-      "-X ${package_url}.gitTag=${tag}"
+      "-X ${package_url}.gitCommit=unknown"
+      "-X ${package_url}.gitTag=v${version}"
       "-X ${package_url}.gitTreeState=clean"
     ];
 
   # Test is disabled because ksonnet is missing from nixpkgs.
   # Log: https://gist.github.com/superherointj/79cbdc869dfd44d28a10dc6746ecb3f9
+  # TODO Remove it in next release
+  # https://github.com/argoproj/argo-cd/commit/df2d6497513630f873dc17452f5ad05aac4188d0
   doCheck = false;
```

Comment:
```suggestion
```
As tests are working now, remove the comments and `doCheck` notation. By default, it will work.

Reply:
Not needed with cmd, also the drop of ksonnet did not have a release yet.